Swift/ConflictResolver.swift

```diff
@@ -20,9 +20,38 @@
 import Foundation
 
 /// Conflict Resolver protocol
-public protocol ConflictResolver {
+public protocol ConflictResolverProtocol {
     
     /// The callback resolve method, if conflict occurs.
     func resolve(conflict: Conflict) -> Document?
     
 }
+
+
+/// Conflict Resolution class
```

Comment:
Maybe `ConflictResolver provides access to the default conflict resolver used by the replicator`.